MovingBlocks/Terasology
# Broadcast games on LAN

### Contains
"Fixes #3760"
New Feature to broadcast games on LAN. This is done by using the  broadcastserver class which sends a broadcast every few seconds and a broadcastclient class which is activated when accessing the LAN servers screen and checks every few seconds for a broadcast and updates the screen .

### How to test

Go to the join game screen and click on the tab for LAN and using another machine on the local network send a broadcast using the start broadcast button on the join game screen to check if the broadcast has been recieved.

### Outstanding before merging



- [x] UI Screen for LAN Servers
- [x] Broadcast on LAN
- [ ] Trigger broadcast when multiplayer game starts
